penguin/integrations/mcp/server_resources.py

```
@@ -0,0 +1,198 @@
+"""MCP resources and prompts for Penguin's MCP server."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Any, Dict, List, Optional
```

Ruff (0.15.12)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)